src/shadowbox/server/manager_service.ts

```diff
@@ -156,6 +163,10 @@ export function bindService(
   );
 
   apiServer.get(`${apiPrefix}/metrics/transfer`, service.getDataUsage.bind(service));
+  apiServer.get(
+    `${apiPrefix}/metrics/tunnel/location`,
```

Comment:
This is not proper REST language. You are retrieving tunnel time, not locations.

I'd call this `/metrics/tunnel_time`.

Also, why breakdown by location only? I suggest we return all the dimensions by default. Or specify the dimensions we want in the request.